gulpfile.js

```diff
@@ -27,7 +27,8 @@ gulp.task('mocha', function () {
   return gulp
     .src(['./test/**/*', '!./test/temp'])
     .pipe(mocha({
-      reporter: 'spec'
+      reporter: 'spec',
+      timeout: Infinity
```

Comment:
I'm indifferent on the solution, but I think that there shouldn't be a timeout at this point. Whether we use timeout(0) or Infinity, it doesn't matter to me.